galsim/chromatic.py

```diff
@@ -162,12 +381,28 @@ def drawImage(self, bandpass, image=None, integrator=None, **kwargs):
                                 integrate.
         @param image            Optionally, the Image to draw onto.  (See GSObject.drawImage()
                                 for details.)  [default: None]
-        @param integrator       One of the image integrators from galsim.integ [default: None,
+        @param integrator       When doing the exact evaluation of the profile, this argument should
+                                be one of the image integrators from galsim.integ [default: None,
                                 which will try to select an appropriate integrator automatically.]
+                                If setupInterpolation() has been called for this object, then
+                                `integrator` should be a string, either 'midpoint' or
+                                'trapezoidal'. [default: 'trapezoidal']
```

Comment:
> But for the normal integration method, I think it's more complicated than this.  Users have to choose one of the integration methods like sample integrator or continuous integrator, and  either the midpoint or trapezoidal method. 

In practice, there is really just 'trapezoid' and 'midpoint' there too.  If the bandpass or sed has a wave_list, then it is pretty ill-advised to use the ContinuousIntegrator, which will rebin the wavelength range into 250 (or N) equally spaced partitions.  It's almost always better to use the actual wavelengths where the sed and/or bandpass are sampled.  And if there is no wave_list for either of them, then it's not possible to use SampleIntegrator.  So in both cases, the only real question is whether to use trapezoid or midpoint rule for the integration.  

But also, I'm not suggesting removing the existing option to provide an integrator object if the user wants.  I think that should still be an option for advanced users who want to do something a bit unusual.  I'm just suggesting that the default be switched from `integrator=None` to `integrator='trapezoidal'` and the code that checks the input `integrator` change to:
```
if isinstance(integrator, str):
    if integrator == 'trapezoidal':
        rule = np.trapz
    elif integrator == 'midpoint'
        rule = galsim.integ.midpt
    if len(wave_list) > 0:
        integrator = galsim.integ.SampleIntegrator(rule)
    else:
        integrator = galsim.integ.ContinuousIntegrator(rule)
```